package.json

```diff
@@ -73,17 +67,19 @@
     "yargs": "11.0.0"
   },
   "peerDependencies": {
-    "mwp-config": "^21.0.2897"
+    "mwp-config": "^21.3.2977-beta"
   },
   "devDependencies": {
-    "babel-eslint": "8.2.3",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
I'm curious what this is for, since `@babel/core` is in regular dependencies?

Reply:
When you run `npx babel-upgrade` it automatically adds it in. I think I can remove this here.